packages/frontend-core/src/fetch/UserFetch.js

```diff
@@ -27,16 +28,25 @@ export default class UserFetch extends DataFetch {
   }
 
   async getData() {
+    const { limit, paginate } = this.options
     const { cursor, query } = get(this.store)
+    let finalQuery
+    // convert old format to new one - we now allow use of the lucene format
+    const { appId, paginated, ...rest } = query
+    if (!LuceneUtils.hasFilters(query) && rest.email) {
+      finalQuery = { string: { email: rest.email } }
+    } else {
+      finalQuery = rest
+    }
```

Comment:
Did you discover where we use this old query? I'd be happy to update those places to use the normal structure now that we support it, but that can be done after in a separate PR to reduce scope of this one.

Reply:
Yeah I'd rather leave the same consistent API to the `UserFetch` for now as fetching by email is nice anyway and really there only is two search options - don't want to expose the "search" until it actually works fully.